Tests/UnitTests/Purchasing/OfferingsManagerTests.swift

```diff
@@ -64,6 +64,7 @@ extension OfferingsManagerTests {
         expect(result).to(beSuccess())
 
         let unwrappedOfferings = try XCTUnwrap(result?.value)
+        expect(unwrappedOfferings.all).to(beEmpty())
```

Comment:
If we add this, would the next check still be necessary?

Reply:
Nope, good point. I think it's still good to leave it to make it explicit that we're also making sure the expected entitlement isn't there.